Jenkinsfile_CNP

```diff
@@ -305,14 +319,15 @@ withPipeline(type, product, component) {
   }
 
   before('functionalTest:preview') {
-    uploadDmnDiagrams()
     // Only for preview upload the definitions via script as high level data set up relies on an existing environment
     uploadCoreCaseDataDefinitions()
   }
 
-  before('functionalTest:aat') {
-    uploadDmnDiagrams()
-  }
+// Enable on aat once Demo testing complete
```

Comment:
Can we also add TODO if we want to make changes in near future? or can be removed, if this is not valid anymore.

Reply:
Yeah added the TODO just now